yew-macro/tests/macro_test.rs

```diff
@@ -23,6 +23,9 @@ fn tests() {
 
     t.pass("tests/macro/html-element-pass.rs");
     t.compile_fail("tests/macro/html-element-fail.rs");
+
+    t.pass("tests/macro/classes-pass.rs");
+    t.compile_fail("tests/macro/classes-fail.rs");
```

Comment:
Please create a new file for the `classes!` macro tests and also move the test files into a dedicated folder (like the `props!` macro).

What's also pretty cool is that you can use a wildcard to select the tests:
```suggestion
    t.pass("tests/classes_macro/*-pass.rs");
    t.compile_fail("tests/classes_macro/*-fail.rs");
```